src/components/molecules/WorldCard/WorldCard.scss

```diff
@@ -0,0 +1,52 @@
+@import "scss/constants";
+
+.WorldCard {
+  display: flex;
+  flex-direction: column;
+  background-color: $secondary--dark;
+  border-radius: $border-radius--lg;
+  margin: $spacing--lg;
+  padding: $spacing--xxl;
+  background-position: center;
+  background-size: cover;
+
+  &__logo {
+    height: 70px;
+    width: 70px;
+    border-radius: 50%;
```

Comment:
```suggestion
    border-radius: $border-radius--max
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2371/commits/cef8d195f3cdcfef440c04428ffe7f938bbb620f